tools/abigen/internal/generate_test.go

```
@@ -86,6 +87,91 @@
+			value = _value;
+		}
+		
+		function getValue() public view returns (uint256) {
+			return value;
+		}
+	}
+	`
+	
+	tempDir, err := os.MkdirTemp("", "solc-test-*")
+	Nil(t, err)
+	defer os.RemoveAll(tempDir)
```

golangci-lint (1.62.2)

Error return value of `os.RemoveAll` is not checked

(errcheck)